starters/vue3-xstate-css/src/views/CounterView.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { useMachine } from '@xstate/vue';
+import ButtonComponent from '../components/ButtonComponent.vue';
+import HeaderComponent from '../components/HeaderComponent.vue';
+import { counterMachine } from '../machines/counterMachine';
+
+const { state, send } = useMachine(counterMachine, { devTools: true });
+</script>
+
+<template>
+  <main>
+    <HeaderComponent>
```

Comment:
For mobile devices, the layout would work better in a column here

<img width="623" alt="Screenshot 2023-04-05 at 12 00 08 PM" src="https://user-images.githubusercontent.com/67210629/230179006-2c8fa320-992e-4bb5-8bde-1b1304c7450a.png">